automation_tools/utils.py

```diff
@@ -1,8 +1,11 @@
 """Utilities tasks and functions"""
 from __future__ import print_function
+from BeautifulSoup import BeautifulSoup
```

Comment:
Last comment, 
1) This line should go before Line10.
2) `from __future__ import print_function`  is an exception as, the whole point of `from __future__ import print_function` is to bring the print function from Python 3 into Python 2.6+.